### PR TITLE
Migrate from intltools to gettext

### PR DESCRIPTION
**Pull request by yetist**
Note: ru and uk languages has errors.

### CODE REVIEW

**Comment by raveit65** [comment]
@monsta
Can you please this translation format issue at transifex for master and 1.22 ?
+ fixing `sl` language.
........and bang the head of Russian translator who are including this errors, .........not the first time.
I did that minimum 7-8 times, but now I am really tired of fixing languages which i don't understand.

**Comment by raveit65** [comment]
`ru` and `uk` are disabled temporarily. Now we can work on PR and @monsta or any other have all the time of the world to fix formating error in translations.

**Comment by raveit65** [comment]
@yetist 
For what is `libappstream-glib-devel` dependency needed?
I never used this for building.

File: Makefile.am [hunk 1]
```diff
@@ -4,7 +4,7 @@ if ENABLE_JPEG
 jpeg_DIRS = jpegutils
 endif
 
-SUBDIRS = $(jpeg_DIRS) cut-n-paste src man plugins po help data doc
+SUBDIRS = po $(jpeg_DIRS) cut-n-paste src man plugins help data doc
```
Comment (raveit65) [inline Makefile.am]: Why sorting `po` to first position?

**Comment by raveit65** [comment]
@yetist 
`diff -uprN eom-master.pot eom.pot > diff-pot.patch`
```
<cut>
 -#: ../data/eom.desktop.in.in.h:2
-msgid "Browse and rotate images"
+#: data/eom.desktop.in.in:12
+msgid "image;viewer;JPEG;PNG;TIFF;SVG;MATE;photos;browse;thumbnails;rotate;"
 msgstr ""
<cut>
```
We need a translator hint. Keywords shouldn't be translated.

**Comment by sc0w** [comment]
This change fixes russian language, but I don't know how to do that at transifex, I can't find it.

```patch
--- a/po/ru.po
+++ b/po/ru.po
@@ -1284,7 +1284,7 @@ msgid_plural ""
 "the %d selected images to the trash?"
 msgstr[0] ""
 "Действительно хотите переместить\n"
-"выбранное изображение в корзину?"
+"%d выбранное изображение в корзину?"
 msgstr[1] ""
 "Действительно хотите переместить\n"
 "%d выбранных изображений в корзину?"

**Comment by yetist** [comment]
> @yetist
> For what is `libappstream-glib-devel` dependency needed?
> I never used this for building.

appdata.its in libappstream-glib-devel used for ./data/eom.appdata.xml.in translation.

File: Makefile.am (hunk 1 above)
Comment (yetist) [reply Makefile.am]: I think the .po file needs to have been generated before creating the desktop file from the desktop.in template. Is it wrong?

**Comment by yetist** [comment]
>  We need a translator hint. Keywords shouldn't be translated.

Done.

**Comment by sc0w** [comment]
This is the fix for both languages, but I don't know how to fix it at transifex.

Both languages are broken at the same place.

@monsta  Please, can you take a look? this seems to be a transifex issue...

```patch
--- a/po/ru.po
+++ b/po/ru.po
@@ -1284,7 +1284,7 @@ msgid_plural ""
 "the %d selected images to the trash?"
 msgstr[0] ""
 "Действительно хотите переместить\n"
-"выбранное изображение в корзину?"
+"%d выбранное изображение в корзину?"
 msgstr[1] ""
 "Действительно хотите переместить\n"
 "%d выбранных изображений в корзину?"

--- a/po/uk.po
+++ b/po/uk.po
@@ -1245,7 +1245,7 @@ msgid_plural ""
 "the %d selected images to the trash?"
 msgstr[0] ""
 "Справді бажаєте перемістити\n"
-"вибране зображення у смітник?"
+"%d вибране зображення у смітник?"
 msgstr[1] ""
 "Справді бажаєте перемістити\n"
 "%d вибрані зображень у смітник?"

**Comment by monsta** [comment]
This is weird... the translation in `msgstr[0]` shouldn't require `%d`, because `msgid` for only one image doesn't have it:

```

**Comment by monsta** [comment]
In addition, I've just built this PR with `ru` and `uk` enabled in Mint 19.1 (based on Ubuntu 18.04). The build is fine. I can't reproduce the error.

**Comment by monsta** [comment]
Ok, it's even more weird. Nothing is built in `po` subdir:
```
Making all in po
make[3]: Entering directory '/home/monsta/mate/eom/po'
make[3]: Nothing to be done for 'all'.
make[3]: Leaving directory '/home/monsta/mate/eom/po'
```

**Comment by sc0w** [comment]
@monsta in master is ok, the build error is building this PR without https://github.com/mate-desktop/eom/pull/245/commits/6d23023c3e9a1c68b0d0fff1b4b33a5152fe41c1

I think it can be fixed modifying the lines `Plural-Forms:` in both .po files, but I think that lines are handled by transifex

**Comment by sc0w** [comment]
`Plural-Forms` lines:

de
`"Plural-Forms: nplurals=2; plural=(n != 1);\n"`

es
`"Plural-Forms: nplurals=2; plural=(n != 1);\n"`

fr
`"Plural-Forms: nplurals=2; plural=(n > 1);\n"`

ru
`"Plural-Forms: nplurals=4; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n%10<=4 && (n%100<12 || n%100>14) ? 1 : n%10==0 || (n%10>=5 && n%10<=9) || (n%100>=11 && n%100<=14)? 2 : 3);\n"`

uk
`"Plural-Forms: nplurals=4; plural=(n % 1 == 0 && n % 10 == 1 && n % 100 != 11 ? 0 : n % 1 == 0 && n % 10 >= 2 && n % 10 <= 4 && (n % 100 < 12 || n % 100 > 14) ? 1 : n % 1 == 0 && (n % 10 ==0 || (n % 10 >=5 && n % 10 <=9) || (n % 100 >=11 && n % 100 <=14 )) ? 2: 3);\n"`

**Comment by sc0w** [comment]
with these changes the build is ok, but I am not sure, probably `nplurals` must be 2 instead 4, anyway, the problem seems to be inside transifex, they add that lines.
```patch
--- a/po/ru.po
+++ b/po/ru.po
@@ -32,7 +32,7 @@ msgstr ""
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Language: ru\n"
-"Plural-Forms: nplurals=4; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n%10<=4 && (n%100<12 || n%100>14) ? 1 : n%10==0 || (n%10>=5 && n%10<=9) || (n%100>=11 && n%100<=14)? 2 : 3);\n"
+"Plural-Forms: nplurals=4; plural=(n != 1);\n"
 
 #. Translaters: This string is for a toggle to display a toolbar.
 #. * The name of the toolbar is automatically computed from the widgets
diff --git a/po/uk.po b/po/uk.po
index 9a7f536..a09819d 100644
--- a/po/uk.po
+++ b/po/uk.po
@@ -27,7 +27,7 @@ msgstr ""
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Language: uk\n"
-"Plural-Forms: nplurals=4; plural=(n % 1 == 0 && n % 10 == 1 && n % 100 != 11 ? 0 : n % 1 == 0 && n % 10 >= 2 && n % 10 <= 4 && (n % 100 < 12 || n % 100 > 14) ? 1 : n % 1 == 0 && (n % 10 ==0 || (n % 10 >=5 && n % 10 <=9) || (n % 100 >=11 && n % 100 <=14 )) ? 2: 3);\n"
+"Plural-Forms: nplurals=4; plural=(n != 1);\n"
 
 #. Translaters: This string is for a toggle to display a toolbar.
 #. * The name of the toolbar is automatically computed from the widgets

**Comment by monsta** [comment]
Let's not break that line with plurals, it's important for having proper translations for different number of plurals (see https://github.com/linuxmint/mintupdate/issues/72 for some examples).

**Comment by monsta** [comment]
With this PR `make distclean` doesn't clean `.gmo` files and other generated stuff from `po` subdir...

```
monsta@jessica:~/mate/eom$ LC_ALL=C git status po
On branch gettext
Your branch is up to date with 'origin/gettext'.

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	po/Makefile.in.in
	po/Makevars.template
	po/Rules-quot
	po/af.gmo
	po/am.gmo
	po/ar.gmo
	po/as.gmo
	po/ast.gmo
	po/az.gmo
	po/be.gmo
	po/bg.gmo
	po/bn.gmo
	po/bn_IN.gmo
	po/boldquot.sed
	po/br.gmo
	po/bs.gmo
	po/ca.gmo
	po/ca@valencia.gmo
	po/cmn.gmo
	po/crh.gmo
	po/cs.gmo
	po/cy.gmo
	po/da.gmo
	po/de.gmo
	po/dz.gmo
	po/el.gmo
	po/en@boldquot.header
	po/en@quot.header
	po/en_AU.gmo
	po/en_CA.gmo
	po/en_GB.gmo
	po/eo.gmo
	po/eom.pot
	po/es.gmo
	po/es_AR.gmo
	po/es_CL.gmo
	po/es_CO.gmo
	po/es_CR.gmo
	po/es_DO.gmo
	po/es_EC.gmo
	po/es_ES.gmo
	po/es_MX.gmo
	po/es_NI.gmo
	po/es_PA.gmo
	po/es_PE.gmo
	po/es_PR.gmo
	po/es_SV.gmo
	po/es_UY.gmo
	po/es_VE.gmo
	po/et.gmo
	po/eu.gmo
	po/fa.gmo
	po/fi.gmo
	po/fr.gmo
	po/fr_CA.gmo
	po/frp.gmo
	po/fur.gmo
	po/fy.gmo
	po/ga.gmo
	po/gd.gmo
	po/gl.gmo
	po/gu.gmo
	po/ha.gmo
	po/he.gmo
	po/hi.gmo
	po/hr.gmo
	po/hu.gmo
	po/hy.gmo
	po/ia.gmo
	po/id.gmo
	po/ie.gmo
	po/ig.gmo
	po/insert-header.sin
	po/is.gmo
	po/it.gmo
	po/ja.gmo
	po/jv.gmo
	po/ka.gmo
	po/kab.gmo
	po/kk.gmo
	po/km.gmo
	po/kn.gmo
	po/ko.gmo
	po/ks.gmo
	po/ku.gmo
	po/ku_IQ.gmo
	po/ky.gmo
	po/li.gmo
	po/lt.gmo
	po/lv.gmo
	po/mai.gmo
	po/mg.gmo
	po/mi.gmo
	po/mk.gmo
	po/ml.gmo
	po/mn.gmo
	po/mr.gmo
	po/ms.gmo
	po/nb.gmo
	po/nds.gmo
	po/ne.gmo
	po/nl.gmo
	po/nn.gmo
	po/nqo.gmo
	po/nso.gmo
	po/oc.gmo
	po/or.gmo
	po/pa.gmo
	po/pl.gmo
	po/pms.gmo
	po/ps.gmo
	po/pt.gmo
	po/pt_BR.gmo
	po/quot.sed
	po/remove-potcdate.sin
	po/ro.gmo
	po/rw.gmo
	po/sc.gmo
	po/si.gmo
	po/sk.gmo
	po/sl.gmo
	po/sq.gmo
	po/sr.gmo
	po/sr@latin.gmo
	po/stamp-po
	po/sv.gmo
	po/ta.gmo
	po/te.gmo
	po/th.gmo
	po/tk.gmo
	po/tr.gmo
	po/tt.gmo
	po/ug.gmo
	po/ur.gmo
	po/ur_PK.gmo
	po/uz.gmo
	po/vi.gmo
	po/wa.gmo
	po/xh.gmo
	po/yi.gmo
	po/yo.gmo
	po/zh_CN.gmo
	po/zh_HK.gmo
	po/zh_TW.gmo
	po/zu.gmo

nothing added to commit but untracked files present (use "git add" to track)
```
I guess that's why nothing gets built in `po` subdir on the next builds.

File: Makefile.am (hunk 1 above)
Comment (raveit65) [reply Makefile.am]: Ok, good.

**Comment by raveit65** [comment]
.gmo files are needed in tarball with gettext, please read previous PRs for more infos.

**Comment by raveit65** [comment]
This is how it looks in code https://github.com/mate-desktop/eom/blob/master/src/eom-window.c#L3332
```
		if (can_trash) {
			prompt = g_strdup_printf (ngettext("Are you sure you want to move\n"
							   "the selected image to the trash?",
							   "Are you sure you want to move\n"
							   "the %d selected images to the trash?", n_images), n_images);
```
In most po files of languages it looks like this, eg. `ca@valencia` `da`.....
```
#: src/eom-window.c:3333
#, c-format
msgid ""
"Are you sure you want to move\n"
"the selected image to the trash?"
msgid_plural ""
"Are you sure you want to move\n"
"the %d selected images to the trash?"
msgstr[0] ""
"Esteu segur que voleu moure \n"
"la imatge seleccionada a la paperera?"
msgstr[1] ""
"Esteu segur que voleu moure \n"
"les %d imatges seleccionades a la paperera?"
```
```
#: src/eom-window.c:3333
#, c-format
msgid ""
"Are you sure you want to move\n"
"the selected image to the trash?"
msgid_plural ""
"Are you sure you want to move\n"
"the %d selected images to the trash?"
msgstr[0] ""
"Ønsker du at flytte det valgte\n"
"billede til papirkurven?"
msgstr[1] ""
"Ønsker du at flytte de %d\n"
"valgte billeder til papirkurven?"
```
But with `uk` or `ru` languages i looks like this.
```
#: src/eom-window.c:3333
#, c-format
msgid ""
"Are you sure you want to move\n"
"the selected image to the trash?"
msgid_plural ""
"Are you sure you want to move\n"
"the %d selected images to the trash?"
msgstr[0] ""
"Справді бажаєте перемістити\n"
"вибране зображення у смітник?"
msgstr[1] ""
"Справді бажаєте перемістити\n"
"%d вибрані зображень у смітник?"
msgstr[2] ""
"Справді бажаєте перемістити\n"
"%d вибраних зображень у смітник?"
msgstr[3] ""
"Справді бажаєте перемістити\n"
"%d вибраних зображень у смітник?"
```
```
#: ../src/eom-window.c:3269
#, c-format
msgid ""
"Are you sure you want to move\n"
"the selected image to the trash?"
msgid_plural ""
"Are you sure you want to move\n"
"the %d selected images to the trash?"
msgstr[0] ""
"Действительно хотите переместить\n"
"выбранное изображение в корзину?"
msgstr[1] ""
"Действительно хотите переместить\n"
"%d выбранных изображений в корзину?"
msgstr[2] ""
"Действительно хотите переместить\n"
"%d выбранных изображений в корзину?"
msgstr[3] ""
"Действительно хотите переместить\n"
"%d выбранных изображений в корзину?"
```
All languages with only `msgstr[0] ` and `msgstr[1] ` are building fine.
But languages with `msgstr[0] `, `msgstr[1] `, `msgstr[2] ` and `msgstr[3] ` are fail to build.
It seems that gettext only support 1 plural form, or not?

So my question to someone who is speaking one of those languages.
Why 3 plural forms and is this correct?

**Comment by yetist** [comment]
This patch can fix the error:
```patch
diff --git a/po/ru.po b/po/ru.po
index 4193135..3eb091f 100644
--- a/po/ru.po
+++ b/po/ru.po
@@ -1284,7 +1246,7 @@ msgid_plural ""
 "the %d selected images to the trash?"
 msgstr[0] ""
 "Действительно хотите переместить\n"
-"выбранное изображение в корзину?"
+"%d выбранное изображение в корзину?"
 msgstr[1] ""
 "Действительно хотите переместить\n"
 "%d выбранных изображений в корзину?"
@@ -1295,427 +1257,443 @@ msgstr[3] ""
 "Действительно хотите переместить\n"
 "%d выбранных изображений в корзину?"
 
diff --git a/po/uk.po b/po/uk.po
index 9a7f536..cb9b598 100644
--- a/po/uk.po
+++ b/po/uk.po
@@ -1245,7 +1207,7 @@ msgid_plural ""
 "the %d selected images to the trash?"
 msgstr[0] ""
 "Справді бажаєте перемістити\n"
-"вибране зображення у смітник?"
+"%d вибране зображення у смітник?"
 msgstr[1] ""
 "Справді бажаєте перемістити\n"
 "%d вибрані зображень у смітник?"
@@ -1256,7 +1218,7 @@ msgstr[3] ""
 "Справді бажаєте перемістити\n"
 "%d вибраних зображень у смітник?"
```

And I found the plural form for  ru is:
```
nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n%10<=4 && (n%100<10 || n%100>=20) ? 1 : 2);
```
the plural form for  uk is:
```
nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n%10<=4 && (n%100<10 || n%100>=20) ? 1 : 2);
```
see http://docs.translatehouse.org/projects/localization-guide/en/latest/l10n/pluralforms.html

**Comment by sc0w** [comment]
@yetist I said it in https://github.com/mate-desktop/eom/pull/245#issuecomment-552028773

but If I try to do it at transifex, it shows me error at website, because the singular doesn't have the `%d`

And I think is not possible to change the plural forms at transifex

**Comment by yetist** [comment]
This is another patch to fix:
```patch
diff --git a/po/ru.po b/po/ru.po
index 4193135..c1f3b33 100644
--- a/po/ru.po
+++ b/po/ru.po
@@ -1275,7 +1275,6 @@ msgid ""
 msgstr "Не удалось найти корзину для «%s». Удалить изображение навсегда?"
 
 #: ../src/eom-window.c:3269
-#, c-format
 msgid ""
 "Are you sure you want to move\n"
 "the selected image to the trash?"
diff --git a/po/uk.po b/po/uk.po
index 9a7f536..960409a 100644
--- a/po/uk.po
+++ b/po/uk.po
@@ -1236,7 +1236,6 @@ msgstr ""
 "Не можу знайти смітника для \"%s\". Бажаєте назавжди знищити це зображення?"
 
 #: ../src/eom-window.c:3269
-#, c-format
 msgid ""
 "Are you sure you want to move\n"
 "the selected image to the trash?"
```

**Comment by sc0w** [comment]
**eog** have `%d` at `msgstr[0]` in `master` and `ru.po` is ok (tested with `msgfmt -c ru.po`)
```
#: src/eog-window.c:3520
#, c-format
msgid ""
"Are you sure you want to move\n"
"the selected image to the trash?"
msgid_plural ""
"Are you sure you want to move\n"
"the %d selected images to the trash?"
msgstr[0] ""
"Действительно хотите переместить\n"
"%d выбранное изображение в корзину?"
msgstr[1] ""
"Действительно хотите переместить\n"
"%d выбранных изображения в корзину?"
msgstr[2] ""
"Действительно хотите переместить\n"
"%d выбранных изображений в корзину?"
```

**Comment by monsta** [comment]
> .gmo files are needed in tarball with gettext, please read previous PRs for more infos.

No problems with tarballs, problem is that `make distclean` doesn't remove what it should.

**Comment by monsta** [comment]
`"Plural-Forms: nplurals=4; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n%10<=4 && (n%100<12 || n%100>14) ? 1 : n%10==0 || (n%10>=5 && n%10<=9) || (n%100>=11 && n%100<=14)? 2 : 3);\n"`

Ok... so in Russian, plurals for 21, 31, 41, ... are the same as singular (see examples at https://github.com/linuxmint/mintupdate/issues/72). In these cases (`n%10==1 && n%100!=11`) the formula returns 0. So `msgstr[0]` gets selected, but it doesn't have `%d`.

**Comment by monsta** [comment]
Try https://github.com/mate-desktop/eom/pull/246 for another type of fix. I think it should not affect translations for the language with one plural form.

**Comment by raveit65** [comment]
@yetist 
`/* xgettext:no-c-format */`
What does that mean?
Can you explain it a bit please?
Your fix looks more simple than other PR.

**Comment by yetist** [comment]
There are two different way to fix this issue:
1. Add %d into the msgstr[0], like #246 does.
2. Remove c-format comment from po file above the msgid.
After 1 or 2, msgfmt will pass.

@raveit65 

The comment in source code is for the second way, used to generation the po file, and remove the c-format comment.

**Comment by raveit65** [comment]
Ok, personal i think your solution is more simple, but i don't mind.
Let's merge https://github.com/mate-desktop/eom/pull/246, we need to test if this really fix the issue after another pull from transifex.

**Comment by raveit65** [comment]
@monsta 
New eom.pot is uploaded to transifex, can you please update `ru`and uk` translations?

**Comment by raveit65** [comment]
Now, this string needs to translated new for all languages.
`de` is updated.

**Comment by raveit65** [comment]
I tested it locally and building works now after a pull from transifex.
But honestly, i prefer to test @yetist fix for pluma PR, because the translation string is now untranslated in all languages and translators have to translate it again.
Fixing the issue in this way isn't really optimal, ihmo.

off topic on
Btw. there is another issue with `ru` translations in user-guide (help/).
But string is only untranslated until someone fix it at transifex.
```
  GEN    ru/
Warning: Could not merge ru translation for msgid:
To open another image, choose <menuchoice> <guimenu>Image</guimenu><guimenuitem>Open</guimenuitem> </menuchoice> again. <application>Image Viewer</application> opens each image in a new window.
```
and other languages.
```
  GEN    it/
Warning: Could not merge it translation for msgid:
Most actions in <application>Image Viewer</application> can be performed several ways. For example, you can open a file in the following ways:
Warning: Could not merge it translation for msgid:
Before printing you need to set the page settings you would like to use. To do that choose <menuchoice><guimenu>Image</guimenu><guimenuitem>Page Setup</guimenuitem></menuchoice>.
```
offtopic off

**Comment by sc0w** [comment]
@raveit65  

> But honestly, i prefer to test @yetist fix for pluma

russian language without `%d` is wrong , it needs `%d` for all.

 @monsta said  (https://github.com/mate-desktop/eom/pull/245#issuecomment-552859634):

> Ok... so in Russian, plurals for 21, 31, 41, ... are the same as singular

and he is right, I did some investigations, and you can see here how works the russian language with numbrers from 0 to 100 using ngettext and the plural form used at transifex:

[russian_numbers.txt](https://github.com/mate-desktop/eom/files/3838881/russian_numbers.txt)

**Comment by sc0w** [comment]
@yetist we can remove debian patch for intltool at travis?

**Comment by yetist** [comment]
> @yetist we can remove debian patch for intltool at travis?

Yes, done.

**Comment by monsta** [comment]
Russian translation is updated. I don't know Ukrainian so I left it for the translators team.

The changed string actually needs translation only for the languages that have one of the plural forms same as singular (in `msgstr[0]`). You shouldn't see that message in German, for example. For only one image you'll get the `"Are you sure you want to move\n\"%s\" to the trash?"` message.

**Comment by monsta** [comment]
It should be the same in Pluma. The new string is a bugfix for the languages that have one of the plural forms same as singular, the old string is used only for singular.

**Comment by raveit65** [comment]
> It should be the same in Pluma. The new string is a bugfix for the languages that have one of the plural forms same as singular, the old string is used only for singular.

Check this translation string at transifex for other languages. Translation is missing in a lot of translation which are using only one plural from, eg. `fr` or `de`.
And i added german translation back fur singular form at transifex yesterday.
 
@sc0w 
> russian language without %d is wrong , it needs %d for all.

I know :smile: but there is an alternative fix, see https://github.com/mate-desktop/eom/pull/245#issuecomment-552878493

**Comment by sc0w** [comment]
is not a real fix, the translation is wrong and it doesn't fix it

**Comment by raveit65** [comment]
eg, french translation was removed https://github.com/mate-desktop/eom/commit/f620f3ddc9f7b6161b4a15610df076fc0eb8559e#diff-42793ab00dcd448b33776fc86e5eaaf4R1263

**Comment by sc0w** [comment]
@raveit65 in languages with one unique singular (like french) the text is ignored

**Comment by raveit65** [comment]
Again, the singular form was removed in french , german and many others ;)

**Comment by monsta** [comment]
Ok, so Transifex removed the whole translation, even though only one form was changed :confused:

**Comment by monsta** [comment]
Hmm... I've lost "Reload Image" and "Date in statusbar" plugins after this.

**Comment by monsta** [comment]
Also `make distclean` still doesn't remove the generated files from `po` subdir (not just `.gmo` files, there's more generated stuff). Doesn't anyone else have that...?

**Comment by yetist** [comment]
> Ok, so Transifex removed the whole translation, even though only one form was changed

Why is that happening? We can't fix the error locally, then merge the existing translated string and then push it to transifix?

**Comment by raveit65** [comment]
>> Ok, so Transifex removed the whole translation, even though only one form was changed

> Why is that happening? We can't fix the error locally, then merge the existing translated string and then push it to transifix?

It is possible to upload single fixed .mo files via command line (tx command) or at transifex website as admin.
Bildschirmfoto zu 2019-11-13 12-05-00

And @monsta  is admin there.

**Comment by raveit65** [comment]
> Hmm... I've lost "Reload Image" and "Date in statusbar" plugins after this.

Confirmed, and i know why we don't see the plugin in preferences.
```
[root@mother rave]# tail -n6 /usr/lib64/eom/plugins/fullscreen.plugin 
Name=Fullscreen with double-click
Icon=view-fullscreen
Description=Activate fullscreen mode with double-click
Authors=Lucas Rocha <lucasr@gnome.org>
Copyright=Copyright © 2007 Lucas Rocha
Website=http://www.mate-desktop.org/
[root@mother rave]# tail -n6 /usr/lib64/eom/plugins/reload.plugin 
Name=Fullscreen with double-click
Icon=view-fullscreen
Description=Activate fullscreen mode with double-click
Authors=Lucas Rocha <lucasr@gnome.org>
Copyright=Copyright © 2007 Lucas Rocha
Website=http://www.mate-desktop.org/
[root@mother rave]# tail -n6 /usr/lib64/eom/plugins/statusbar-date.plugin 
Name=Fullscreen with double-click
Icon=view-fullscreen
Description=Activate fullscreen mode with double-click
Authors=Lucas Rocha <lucasr@gnome.org>
Copyright=Copyright © 2007 Lucas Rocha
Website=http://www.mate-desktop.org/
```
All plugins use same `Name` an `Icon` entries.
@yetist 
This needs to be fixed.
...restoring branch.

**Comment by yetist** [comment]
@raveit65 

#247 created.

**Pull request merged**